sql/src/main/java/io/crate/execution/jobs/RootTask.java

```diff
@@ -179,29 +178,13 @@ String coordinatorNodeId() {
         return participatedNodes;
     }
 
-    private void prepare(List<Task> orderedTasks) throws Exception {
-        for (int i = 0; i < orderedTaskIds.size(); i++) {
-            int id = orderedTaskIds.get(i);
-            Task task = orderedTasks.get(i);
-            jobsLogs.operationStarted(id, jobId, task.name());
-            try {
-                task.prepare();
-            } catch (Exception e) {
-                for (; i >= 0; i--) {
-                    id = orderedTaskIds.get(i);
-                    jobsLogs.operationFinished(id, jobId, "Prepare: " + SQLExceptions.messageOf(e), -1);
-                }
-                throw e;
-            }
-        }
-    }
-
     public void start() throws Throwable {
         for (IntCursor id : orderedTaskIds) {
             Task task = tasksByPhaseId.get(id.value);
             if (task == null || closed.get()) {
                 break; // got killed before start was called
             }
+            jobsLogs.operationStarted(id.value, jobId, task.name());
```

Comment:
Do we also need to log the operationFinished in case `start` fails? Or do the tasks themselves take care of it?